api/URL.json

```diff
@@ -978,25 +946,76 @@
               "version_added": "54"
             },
             "ie": {
-              "version_added": null
+              "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": true
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": true
             },
             "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "toString": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL/toString",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
This was [added to all three](https://storage.googleapis.com/chromium-find-releases-static/40e.html#40ebec288b976477aaa318b26899dc08f5b15426) flavors in Chrome 52. Note that this doesn't appear in the IDL because `href` [was turned into a stringifier](https://storage.googleapis.com/chromium-find-releases-static/ca9.html#ca9d83f416cb0fc53e886c2bf308a167d3d12758) which gets `toString()` without explicitly naming it.